README.md

```diff
@@ -110,6 +121,22 @@ verifier instead of plain text.
 Optionally specifies the name of the default superuser account. Defaults to
 `edgedb` if not specified.
 
+#### `EDGEDB_GENERATE_SELF_SIGNED_CERT`
+
+Set this option to `1` to tell the server to automatically generate a
+self-signed certificate with key file in the `EDGEDB_DATADIR` (if present,
+see below), and echo the certificate content in the logs. If the certificate
+file exists, the server will use it instead of generating a new one.
+
+Self-signed certificates are usually used in development and testing, you
+should likely provide your own certificate and key file with the variables below.
+
+#### `EDGEDB_CERT_FILE`, `EDGEDB_KEY_FILE`
```

Comment:
Let's use EDGEDB_TLS_CERT_FILE and EDGEDB_TLS_KEY_FILE for consistency with command line args

Reply:
Sure thing!